content/general/concepts/hexadecimal/hexadecimal.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: 'Hexadecimal'
+Description: 
+Subjects: 
+    - 'Web Development'
+    - 'Web Design'
+    - 'Computer Engeneering'
+Tags:
+    - 'color'
```

Comment:
Tags need to come form the list in the [`tags.md` file in the `/documentation` folder.](https://github.com/Codecademy/docs/blob/main/documentation/tags.md)